### PR TITLE
chore(ui): add marker and refresh icons

### PR DESCRIPTION
**Pull request by gtarpenning**
add marker and refresh icons.

### CODE REVIEW

File: weave-js/src/assets/icons/icon-reload-refresh.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
+<path d="M14.9899 5.39314C14.4497 5.1492 13.8703 4.96797 13.2589 4.86016C13.045 4.82243 12.8315 4.7945 12.619 4.77605C12.415 4.7588 12.2085 4.75 12 4.75C10.2204 4.75 8.59061 5.39116 7.32895 6.45508C6.51641 7.14027 5.85656 8.00081 5.40775 8.97836C5.1563 9.52793 4.96998 10.1182 4.86014 10.7411C4.85781 10.7543 4.85514 10.7674 4.85215 10.7803C4.78498 11.1769 4.75 11.5844 4.75 12C4.75 16.0041 7.99594 19.25 12 19.25C16.0041 19.25 19.25 16.0041 19.25 12C19.25 11.5858 19.5858 11.25 20 11.25C20.4142 11.25 20.75 11.5858 20.75 12C20.75 16.8325 16.8325 20.75 12 20.75C7.16751 20.75 3.25 16.8325 3.25 12C3.25 10.6987 3.53405 9.46383 4.04353 8.35388C5.13014 5.98658 7.24218 4.18774 9.81091 3.52607C10.752 3.28234 11.7461 3.19314 12.7588 3.28246C13.8001 3.37193 14.7891 3.64379 15.6939 4.06579C16.2943 4.34488 16.8542 4.6886 17.3663 5.08671C17.6794 5.33013 17.9745 5.59382 18.25 5.87542V3.75C18.25 3.33579 18.5858 3 19 3C19.4142 3 19.75 3.33579 19.75 3.75V8C19.75 8.41421 19.4142 8.75 19 8.75L14.75 8.74999C14.3358 8.74999 14 8.4142 14 7.99999C14 7.58578 14.3358 7.24999 14.75 7.24999L17.4775 7.25C16.7958 6.46459 15.9484 5.82757 14.9899 5.39314Z" fill="#2B3038"/>
```
Comment (jwlee64) [inline weave-js/src/assets/icons/icon-reload-refresh.svg:2]:
```suggestion
<path d="M14.9899 5.39314C14.4497 5.1492 13.8703 4.96797 13.2589 4.86016C13.045 4.82243 12.8315 4.7945 12.619 4.77605C12.415 4.7588 12.2085 4.75 12 4.75C10.2204 4.75 8.59061 5.39116 7.32895 6.45508C6.51641 7.14027 5.85656 8.00081 5.40775 8.97836C5.1563 9.52793 4.96998 10.1182 4.86014 10.7411C4.85781 10.7543 4.85514 10.7674 4.85215 10.7803C4.78498 11.1769 4.75 11.5844 4.75 12C4.75 16.0041 7.99594 19.25 12 19.25C16.0041 19.25 19.25 16.0041 19.25 12C19.25 11.5858 19.5858 11.25 20 11.25C20.4142 11.25 20.75 11.5858 20.75 12C20.75 16.8325 16.8325 20.75 12 20.75C7.16751 20.75 3.25 16.8325 3.25 12C3.25 10.6987 3.53405 9.46383 4.04353 8.35388C5.13014 5.98658 7.24218 4.18774 9.81091 3.52607C10.752 3.28234 11.7461 3.19314 12.7588 3.28246C13.8001 3.37193 14.7891 3.64379 15.6939 4.06579C16.2943 4.34488 16.8542 4.6886 17.3663 5.08671C17.6794 5.33013 17.9745 5.59382 18.25 5.87542V3.75C18.25 3.33579 18.5858 3 19 3C19.4142 3 19.75 3.33579 19.75 3.75V8C19.75 8.41421 19.4142 8.75 19 8.75L14.75 8.74999C14.3358 8.74999 14 8.4142 14 7.99999C14 7.58578 14.3358 7.24999 14.75 7.24999L17.4775 7.25C16.7958 6.46459 15.9484 5.82757 14.9899 5.39314Z" fill="#currentColor"/>
```

Comment (jwlee64) [reply weave-js/src/assets/icons/icon-reload-refresh.svg:2]: You have to update the fills to currentColor and and the icons to index.ts in /Icon, I suggest using jamies addIcon script in devtools repo

Comment (gtarpenning) [reply weave-js/src/assets/icons/icon-reload-refresh.svg:2]: Thanks!

**Review by jwlee64 — APPROVED** [review]
noice

**Pull request merged**